index.html

```diff
@@ -17,32 +17,32 @@ <h2>IdeaBox</h2>
     <div class="wrapper">
       <form class="custom-form">
         <label for="titleInput">Title</label>
-        <input class="user-input title-input" id="titleInput"></input>
+        <input class="user-input title-input" id="titleInput" aria-required="true"></input>
         <label for="bodyInput">Body</label>
-        <input class="user-input body-input" id="bodyInput"></input>
-        <button class="save-button" id="saveButton">Save</button>
+        <input class="user-input body-input" id="bodyInput" aria-required="true"></input>
+        <button class="save-button" id="saveButton" aria-label="Save Button">Save</button>
```

Comment:
When it comes to aria labels, buttons are semantic elements already, so adding an aria label may just overcomplicate the HTML and actually hinder accessibility. As long as buttons have an innerText they should be picked up as such by a screenreader etc. I would remove the aria labels for buttons and just double check that they all have appropriate inner text